Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Boo.Lang;
```

Comment:
I'm going to add a script to make sure we don't have any instances of Boo.Lang anymore.

As much as I love Boo:

![image](https://user-images.githubusercontent.com/5840182/62971853-9e179d00-bdc7-11e9-928c-abf3f6ecc60a.png)

Reply:
This should probably be System.Collections

Reply:
agreed. i'll fix

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5632